mappings/bnfdmd/README

```diff
@@ -0,0 +1,23 @@
+BNF to DM+D (SNOMED) Mappings
+------------------------------------------
+
+dm+d codes are used to identify medicines and devices in use across the NHS.
+
+However, dm+d codes are not very human-readable, so codelists are typically
+build using pseudoBNF and converted to dm+d for use in study definitions.
```

Comment:
As I understand it, the main reason for constructing a BNF codelist and converting it to dm+d is that BNF has a richer structure than dm+d.

Reply:
Brian said this in the blog post, but maybe I'll just refer to that :)